staging/src/k8s.io/apimachinery/pkg/runtime/error.go

```diff
@@ -120,3 +120,77 @@ func IsMissingVersion(err error) bool {
 	_, ok := err.(*missingVersionErr)
 	return ok
 }
+
+// StrictDecoderError is a base error type that is returned by a strict Decoder such
```

Comment:
the @neolit123 in the title will spam me on each rebase or fork.
could you please omit the `@`?

Reply:
ok, I'll do so next push